public/src/js/pages/home.ts

```diff
@@ -51,14 +51,18 @@ export class HomePage extends Page {
 
   private createEventDom(event: Event) {
     const ele = RenderTemplate('event', null, event);
+    const user = firebase.auth().currentUser!;
+    if (event.host === user.uid) {
+      ele.append('<div class="ribbon ribbon-top-right"><span>Host</span></div>')
```

Comment:
can we make a ribbon template?

then you could do something like:

ele.append(RenderTemplate('ribbon', null, {'title': 'Host'}}))

or something like that.